src/applications/caregivers/helpers.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+export const directDepositWarning = (
+  <div className="pension-dd-warning">
+    The Department of Treasury requires all federal benefit payments be made by
+    electronic funds transfer (EFT), also called direct deposit. If you don’t
+    have a bank account, you must get your payment through Direct Express Debit
+    MasterCard. To request a Direct Express Debit MasterCard you must apply at{' '}
+    <a
+      href="http://www.usdirectexpress.com"
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      www.usdirectexpress.com
+    </a>{' '}
+    or by telephone at{' '}
+    <a href="tel:8003331795" target="_blank" rel="noopener noreferrer">
```

Comment:
I think `target="_blank"` doesn't do anything for phone number a tags? But not 100% sure though!

Reply:
You are defiantly right good catch!